src/api-binder/report.ts

```diff
@@ -1,5 +1,6 @@
 import * as url from 'url';
 import * as _ from 'lodash';
+import { delay } from 'bluebird';
```

Comment:
Can we use `sleep` from backoff.ts? (There's also a sleep function in test/lib/helpers.ts but we previously agreed to prefer `sleep` from backoff over helpers)

Reply:
I wanted to and checked because I recalled this too but the sleep function in backoff isn't exported so I just did what target state fetching does by using bluebird delay instead of introducing an export sleep function when maybe we should try to just upgrade to node 16 for native setTimeout promise